docs/howto/build-run.md

```diff
@@ -268,6 +280,23 @@ If you've run `react-native link`, you can discard the edits it made
 (along with any other edits you've made) by running `git reset --hard`.
 
 
+### Build failure: missing `/node_modules/react-native/third-party/double-conversion-1.1.6/src/fast-dtoa.cc`
+
+When trying to build the iOS app, if you get an error like
+
+```
+error: Build input file cannot be found: '/Users/jappleseed/dev/zulip-mobile/node_modules/react-native/third-party/double-conversion-1.1.6/src/bignum.cc' (in target 'double-conversion' from project 'React')
+```
+
+then try restarting Xcode.
+
+
+### App gets stuck on "Downloading JavaScript bundle 100%" with no loading spinner
+
+Try using the "shake" gesture and turning off remote debugging. (You can turn it
+on later.)
+
```

Comment:
Huh interesting. I think I see this regularly, and just reload -- i.e., use the "shake" gesture and choose reload there, or just hit `r` `r` in the Android emulator. (Or IIRC Cmd+R in the iOS simulator.)

One side effect of turning remote debugging off, or on, is that it'll cause a reload. So I kind of suspect that that's what's making this tip work. Is that consistent with what you've seen?

Reply:
No, reloading wasn't working for me. I came across this solution from Stack Overflow; see https://stackoverflow.com/a/54318120/6792075.